support-frontend/assets/helpers/page/page.ts

```diff
@@ -14,29 +14,35 @@ import {
 import { getReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 import { getSettings } from '../globalsAndSwitches/globals';
 
-function setUpTrackingAndConsents(): void {
+function setUpTrackingAndConsents(participations: Participations): void {
+	const countryId: IsoCountry = Country.detect();
+	const acquisitionData = getReferrerAcquisitionData();
+
+	void consentInitialisation(countryId);
+	analyticsInitialisation(participations, acquisitionData);
+	sendConsentToOphan();
+}
+
+function getAbParticipations(): Participations {
 	const settings = getSettings();
 	const countryId: IsoCountry = Country.detect();
 	const countryGroupId: CountryGroupId = CountryGroup.detect();
 	const abtestInitalizerData = {
 		countryId,
 		countryGroupId,
 	};
+
 	const participations: Participations = abTest.init(abtestInitalizerData);
```

Comment:
So this is now the only place this is called?

Reply:
It's now called from 2 places:
1. here (for non-redux pages)
2. the place where the redux store is initialised